internal/core/window.rs

```diff
@@ -222,6 +222,11 @@ pub trait WindowAdapterInternal {
     {
         Err(raw_window_handle_06::HandleError::NotSupported)
     }
+
+    /// Brings the window to the front and focuses it.
+    fn focus_window(&self) -> Result<(), PlatformError> {
```

Comment:
I'm wondering about the name. This does more than focusing the window, it also bring it to the top. And is there really the need to repeat `_window` when this is called on the window adapter? (also, focus_window seem like a getter)

Maybe i'd go with `bring_to_front()`  what do you think?

Reply:
Sounds, good, `bring_to_front()` is indeed much better.